app/controllers/api/v1/npq_profiles_controller.rb

```diff
@@ -4,13 +4,54 @@ module Api
   module V1
     class NPQProfilesController < Api::ApiController
       include ApiTokenAuthenticatable
-      include JSONAPI::ActsAsResourceController
+
+      def create
+        validation_data = NPQValidationData.new(validation_data_params)
+        validation_data.npq_course = NPQCourse.find(npq_course_param)
+        validation_data.npq_lead_provider = NPQLeadProvider.find(npq_lead_provider_param)
+        validation_data.user = User.find(user_param)
+
+        if validation_data.save
+          render status: :created,
+                 content_type: "application/vnd.api+json",
+                 json: NPQValidationDataSerializer.new(validation_data).serializable_hash
+        end
```

Comment:
If it is not saved, do we want to surface validation errors?

Reply:
It's in the next commit. Sorry, I saw your message after I made that commit.